apps/web/pages/export.tsx

```diff
@@ -35,11 +33,17 @@ interface ServerSideProps {
 export const getServerSideProps: GetServerSideProps<ServerSideProps> = async (
   ctx,
 ) => {
-  const {serverGetUser} = await import('../server')
   const user = await serverGetUser(ctx)
+  if (!user?.id) {
+    return {
+      redirect: {
+        destination: '/auth',
+        permanent: false,
+      },
+    }
+  }
```

Comment:
Come to think of it. We need this server-side even when we have client-side redirect otherwise zod will fail when parsing `apiKey`.